includes/plugins/wc-memberships/class-memberships.php

```diff
@@ -78,6 +80,13 @@ public static function init() {
 		include __DIR__ . '/class-metering.php';
 	}
 
+	/**
+	 * Check if Memberships is available.
+	 */
+	public static function has_memberships() {
```

Comment:
Nit: the method name suggests to me that it checks if an entity (site/user) has "memberships", not whether a plugin is active.

Reply:
Good point—01d8e275 updates this to `is_active()` instead.